app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -353,7 +354,7 @@ class UrlInputFragment :
 
         binding.browserToolbar.editMode()
         setHomeMenu()
-        if (Features.ARE_HOME_PAGE_PRO_TIPS_ENABLED) {
+        if (!FocusNimbus.features.onboardingWithCfrs.value().isOnboardingWithCfrs) {
```

Comment:
The feature here: `onboardingWithCfrs` doesn't fit with what it replaced: `ARE_HOME_PAGE_PRO_TIPS_ENABLED`.

Perhaps this needs to be in a feature grouped together with the FML equivalent of `SEARCH_TERMS_OR_URL`.

Reply:
In our case cfrs were designed to replace home screen tips, so if the experiment is not enabled we should follow the old path